examples/ui.content/src/main/content/jcr_root/conf/core-components-examples/settings/wcm/policies/.content.xml

```diff
@@ -3,100 +3,30 @@
           xmlns:jcr="http://www.jcp.org/jcr/1.0" xmlns:nt="http://www.jcp.org/jcr/nt/1.0" xmlns:rep="internal"
           jcr:mixinTypes="[rep:AccessControllable]"
           jcr:primaryType="cq:Page">
-    <core-components-examples jcr:primaryType="nt:unstructured">
```

Comment:
why are the existing policies removed? these are required for css in the examples site page (aemcomponents.dev).
Also, form container policy is required for allowing insertion of forms core components in the examples page.